valohai_yaml/schema/pipeline.yaml

```diff
@@ -38,6 +38,13 @@ properties:
           required:
             - source
             - target
+  allow-reuse:
+    type: boolean
+    default: false
+    description: |
+      Set to true to allow Valohai to automatically detect if the executions in
+      steps' nodes could be skipped and reuse the result from a previously successfully
+      run node.
```

Comment:
Somewhere in the description, we should mention the conditions that lead to execution being matched (i.e., that the execution is in the same project, that it has the same inputs, parameters, code commit, and data).

Reply:
Could you suggest some text for it? Tbh I'm not good with the wording 😂

Reply:
added some more text to it 👍